packages/snap-preact/src/Templates/Stores/ThemeStore.tsx

```diff
@@ -274,25 +267,20 @@ export function mergeThemeLayers(...layers: ThemePartial[]): ThemePartial {
 	return deepmerge.all(layers, { arrayMerge: arrayMerge });
 }
 
-export function setActiveBreakpoint(width: number | undefined, breakpoints: ThemeVariableBreakpoints | undefined): ThemePartial {
+export function getActiveBreakpoint(width: number | undefined, breakpoints: ThemeVariableBreakpoints | undefined): ResponsiveKeys {
 	let breakpoint: ResponsiveKeys | undefined;
 
 	if (Number.isInteger(width) && breakpoints) {
 		breakpoint = Object.keys(breakpoints).find((breakpoint) => width! <= breakpoints[breakpoint as keyof typeof breakpoints]) as ResponsiveKeys;
 	}
-	return { activeBreakpoint: breakpoint || 'default' };
+	return breakpoint || 'default';
 }
 
-export function getOverridesAtWidth(width: number | undefined, breakpoints: ThemeVariableBreakpoints | undefined, theme: ThemePartial): ThemePartial {
+export function getOverridesAtActiveBreakpoint(activeBreakpoint: ResponsiveKeys, width: number | undefined, theme: ThemePartial): ThemePartial {
```

Comment:
It doesn't seem the width is needed in this function any longer... can it be removed?

Reply:
well I removed it initially and tests broke. certain tests have innerwidth set to 0 which made these return with empty prop overrides. after my changes they were getting some responsive overrides that they were not getting previously. I assumed those tests were there for good reason 🤷 so I added the width back in to keep the logic consistent as it was before.

Reply:
The functionality and interface of the function has changed, so I think it is expected that the tests should change as well (as opposed to keeping parameters that have no function).